k8s/production/runners/protected/x86_64/v3/release.yaml

```diff
@@ -63,12 +63,39 @@ spec:
             poll_timeout = 600  # ten minutes
             service_account = "runner"
 
-            # Place pod close to other pipeline pods if possible ("pack" the pods tightly)
-            # Docs: https://docs.gitlab.com/runner/executors/kubernetes.html#define-nodes-where-pods-are-scheduled
             [runners.kubernetes.affinity]
+              [runners.kubernetes.affinity.node_affinity]
+
+              # Schedule this pod on any node with x86_64 >= v3
```

Comment:
Why does the signing runner need x86_64 >= v3? Is it incompatible with v2?

Reply:
It was scheduled in the x86-64-v3 pool prior to this, so I was just ensuring that this change doesn't break compatibility.